core/domain/exp_domain_test.py

```diff
@@ -4459,17 +4606,26 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
     content:
       content_id: content
       html: <p>Hello, this is state3</p>
+    image_assets:
+      image_mapping:
+        \'1\':
```

Comment:
are we expecting the image_id to be a string? (somewhat like `image-1`.)

I'm leaving the review here can you please make sure that the image id type is consistent in frontend, backend and datastore?

Reply:
sorry, for it.
I have done all the changes
Thanks